src/views/Random.vue

```diff
@@ -8,7 +8,7 @@
           Random image
         </h1>
         <p class="mt-1 max-w-2xl text-sm text-gray-500 dark:text-gray-400">
-          Here's a random image from the collections.
+          Here's a random image from any collection.
```

Comment:
```suggestion
          Here's a random image from one of the collections.
```
I think this flows a bit weirdly. Maybe something like this?